changes/v5.0.10-en.md

```diff
@@ -29,3 +29,5 @@
   This bug was introduced in v5.0.9. For Rule-Engine's input events like `$events/message_delivered`
   and `$events/message_dropped`, if the message was delivered to a shared-subscription,
   the encoding (to JSON) of the event will fail.
+
+- Fix unexpected http status code 400 instead of 404 in '/gateways' API [#9268](https://github.com/emqx/emqx/pull/9268)
```

Comment:
```suggestion
- Fix bad HTTP response status code for `/gateways` API, when Gateway name is unknown, it should return `404` instead of `400` [#9268](https://github.com/emqx/emqx/pull/9268).
```